File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailyCommunicationTask.ts
import { CommunicationSetup } from "../AVCommunications/communicationSetup";
import { DailyBaseTask } from "./dailyBaseTask";

export class DailyCommunicationTask extends DailyBaseTask{    
    communicationSetupId: number;
    communicationSetup: CommunicationSetup;    
}

File: FMAplication/Domain/DailyTasks/DailyCommunicationTask.cs
using System;
using FMAplication.Core;
using FMAplication.Domain.AVCommunications;
using FMAplication.Domain.Bases;
using FMAplication.Domain.Sales;
using FMAplication.Enumerations;

namespace FMAplication.Domain.DailyTasks
{
    public class DailyCommunicationTask : DailyBaseTask
    {
        
        public int CommunicationSetupId { get; set; }

    }
}